src/modules/infobox/components/styling/DrawingStylePopoverButton.vue

```diff
@@ -1,6 +1,10 @@
 <template>
     <div>
-        <button ref="popoverButton" class="btn btn-sm btn-light d-flex align-items-center">
+        <button
+            ref="popoverButton"
+            class="btn btn-sm btn-light d-flex align-items-center"
+            :class="[buttonClassOptions ? buttonClassOptions : '']"
+        >
```

Comment:
For the closing action it might make sense and be easier to handle the show/hide in this component by using a `show` data that is used to display the popover content
```
<div v-if="show" class="popover-content card-body rounded-bottom">
    <slot />
</div>

```

This way in you slot component you can listen to onMounted event to clear its inputs

Reply:
As discussed together, due to code complexity, no need to clear the form when closing it.